features/step_definitions/node.rb

```diff
@@ -683,3 +683,8 @@
    admin.cli_exec(:oadm_cordon_node, opts)
   end
 end
+
+Given /^I store the schedulable workers without taints in the :nodes clipboard$/ do 
```

Comment:
@ropatil010 generally looks good...one area of improvement is to make the clipboard name to be a variable as well...not a deal breaker...just a suggestion from my end.

Reply:
Thanks @pruan-rht. Fixed as per your suggestion.
Logs:  https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/1024395/console 

08-29 12:43:36.311  1 scenario (1 passed)